owncloudApp/src/main/java/com/owncloud/android/ui/activity/LogHistoryActivity.java

```diff
@@ -131,28 +131,21 @@ public boolean onCreateOptionsMenu(Menu menu) {
         SearchManager searchManager = (SearchManager) getSystemService(Context.SEARCH_SERVICE);
 
         if (searchItem != null) {
-            searchView = (SearchView) searchItem.getActionView();
-            //            searchView.setQueryHint(getString(R.string.text));
+            mSearchView = (SearchView) searchItem.getActionView();
         }
-        if (searchView != null) {
-            searchView.setSearchableInfo(searchManager.getSearchableInfo(getComponentName()));
-        }
-
-        if (searchView == null) {
```

Comment:
If you remove this null check it could end up with null pointer exceptions when using `mSearchView.setSearchableInfo` or even `mSearchView.setIconified(false);` so I would keep it as it is and remove the coming null checks over mSearchView.